core/evancore.vue.libs/src/components/table/table.scss

```diff
@@ -145,6 +152,13 @@ $arrow-up: '<svg xmlns="http://www.w3.org/2000/svg" style="width:24px;height:24p
     .table-spacer {
       height: 16px;
     }
+
+    .list-enter-active, .list-leave-active {
```

Comment:
where are those classes used?

Reply:
They are "magically" attached by vue's built in transition (which is used by vue bootstrap table internally)